protocol/src/main/java/io/camunda/zeebe/protocol/record/intent/ResourceDeletionIntent.java

```diff
@@ -39,6 +40,8 @@ public static Intent from(final short value) {
       case 0:
         return DELETE;
       case 1:
+        return DELETING;
+      case 2:
```

Comment:
:question: Was this previously released? If it was in previous versions, this could cause breaking changes no?

Reply:
The feature was disabled in previous versions as it wasn't ready yet. So there is no chance that an event with this Intent could have been written before.

So yes, normally it would be breaking. In that case it isn't and I liked this order better 🤷